entries/l/letsdothis.com.json

```
@@ -0,0 +1,16 @@
+{
+  "LetsDoThis": {
```

The official name seems to be this:
```suggestion
  "Let's Do This": {
```